modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/tree/BplusTree.java

```diff
@@ -113,8 +113,8 @@
  * </code></pre>
  * There are {@code N} items and {@code N+1} links. Each link points to page of a lower level. For example, pages on level {@code 2} always
  * point to pages of level {@code 1}. For an item {@code i} left subtree is defined by {@code link(i)} and right subtree is defined by
- * {@code link(i+1)} ({@link BplusInnerIo#getLeft(long, int)} and {@link BplusInnerIo#getRight(long, int)}). All items in the left subtree
- * are less or equal to the original item (basic property for the trees).
+ * {@code link(i+1)} ({@link BplusInnerIo#getLeftPageId(long, int, int)} and {@link BplusInnerIo#getRightPageId(long, int, int)}).
```

Comment:
Same for these two. Please also return the old formatting back after renaming, if it fits

Reply:
Fix it